site/content/docs/tutorials/concierge-and-supervisor-demo.md

```diff
@@ -1,239 +1,654 @@
 ---
-title: Learn to use the Pinniped Supervisor alongside the Concierge
+title: Learn to use Pinniped for federated authentication to Kubernetes clusters
 description: See how the Pinniped Supervisor streamlines login to multiple Kubernetes clusters.
 cascade:
   layout: docs
 menu:
   docs:
     name: Concierge with Supervisor
     parent: tutorials
+    weight: 1
 ---
 
-## Prerequisites
+## Why Pinniped?
+
+There are many benefits to using the Pinniped Supervisor, Concierge, and CLI components together
+to provide Kubernetes authentication.
```

Comment:
We could likely come up with more benefits, but I think this list is great as-is.

Reply:
1. Jumpbox support
2. Hardened TLS
3. ...

Reply:
I had also considered adding jumpbox support to the list, but then I was worried that it would turn it into a list of *features* instead of a list of *benefits*.

I wasn't exactly sure where to draw the line. I felt like maybe jumpbox support is a feature which exists to support this benefit:

> Deep integration with `kubectl` means that when a user runs `kubectl` commands, they will be **interactively prompted to log in using their own unique identity** from your identity provider.

Maybe hardened TLS could be described as benefit something along the lines of being "secure by default, wherever  possible"?

Reply:
I think the existing list is great as-is 😄